doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
```

Comment:
Assuming we're not implementing multiple repositories per project, the association `changeset - repository` and `changeset - project` is symmetric. If we decide to that a repository endpoint is required, we can link it **additionally** here.

Still, we can think about whether a repository endpoint would make sense. It could provide the following features:

* Branches
* Tags
* File access ? 
* Diff ?